src/styles/_global.scss

```diff
@@ -1,6 +1,6 @@
 .add-feedback {
     align-items: center;
-    background-color: $violet;
+    background-color: $violet !important;
```

Comment:
We can tackle the css again, I think we can create a `css cleanup` PR to remove some of these `!importants`.

We should really try to avoid `!important` at all costs because it makes overrides and tracking things down an absolute nightmare.

Reply:
Ah ok. I'll make the PR for that next.